views/components/user-navigation/macro.njk

```diff
@@ -13,26 +13,12 @@
                     {% if previousPage %}
                         <li class="user-nav__pagination-link user-nav__pagination-link--prev">
                             <span class="user-nav__pagination-arrow">←</span>
-                            <a href="{{ previousPage.url }}">
-                                <span>
-                                    {{ __('applyNext.navigation.previous') }}
-                                </span>
-                                <span class="user-nav__pagination-subtitle">
-                                    {{ previousPage.label }}
-                                </span>
-                            </a>
+                            <input class="btn-link" name="previousBtn" type="submit" value="{{ __('applyNext.navigation.previous') }}"/>
```

Comment:
Not something for now, but I'd like to revisit how this macro is structured as it currently does two things: form pagination and account links. The account link functionality could be pulled out. Currently this macro is used on pages that aren't forms so has location dependent behaviour which I'd like to fix. Your change here looks good though 👍